CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -141,7 +141,11 @@ end
 local function SetTransparency(transparency)
 	for i, v in pairs(Tools) do
 		v.icon.BackgroundTransparency = transparency + 0.5
-		v.icon.ImageTransparency = transparency
```

Comment:
was this failing before? would rather not do string compare unless it is necessary.  We should probably use our API for this if we have to, you can call v.icon:IsA